MobiFlight-Installer/MobiFlight-Installer/UI/InstallMainForm.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Net;
+using System.Net.Http;
+
+namespace MobiFlightInstaller.UI
+{
+    public partial class InstallMainForm : Form
+    {
+        private string _downloadURL = "";
+        private string _downloadChecksum = "";
+        private WebClient _webClient;
+        private string CurrentFileName = "";
+        private string PathToInstall = "";
+
+        public InstallMainForm()
+        {
+            InitializeComponent();
+            SelectedPath.Text = Directory.GetCurrentDirectory();
+            AsyncCheckGithub();
+            BoxTargetDirectory.Checked = MobiFlightUpdaterModel.VerifyCurrentFolderRight();
+            IfReadyForTakeOFF();
+        }
+
+        private void button1_Click(object sender, EventArgs e)
```

Comment:
The buttons (both `button1` and `button2`) should be given real names so it's clear what their function is when reading the code, and the event handlers should use those names. This button looks like it's for selecting install location? something like `installLocationButton` would work for example, and the handler would be `installLocationButton_click()`.